internal/acceptance/openstack/identity/v3/projects_test.go

```diff
@@ -393,3 +393,23 @@ func TestProjectsTagsCRUD(t *testing.T) {
 	err = projects.DeleteTags(context.TODO(), client, projectMain.ID).ExtractErr()
 	th.AssertNoErr(t, err)
 }
+
+func TestTagCRUD(t *testing.T) {
+	clients.RequireAdmin(t)
+
+	client, err := clients.NewIdentityV3Client()
+	th.AssertNoErr(t, err)
+
+	projectMain, err := CreateProject(t, client, nil)
+	th.AssertNoErr(t, err)
+	defer DeleteProject(t, client, projectMain.ID)
+
+	err = projects.AddTag(context.TODO(), client, projectMain.ID, "Tag1").ExtractErr()
+	th.AssertNoErr(t, err)
+
+	err = projects.CheckTag(context.TODO(), client, projectMain.ID, "Tag1").ExtractErr()
+	th.AssertNoErr(t, err)
+
+	err = projects.DeleteTag(context.TODO(), client, projectMain.ID, "Tag1").ExtractErr()
```

Comment:
Nice test sequence! It would be nice to run `CheckTag` here below to verify that it's gone.

Reply:
Added the extra check tag at the end.